### PR TITLE
[RTC-39] Add vadNotification Media Event

### PR DESCRIPTION
**Pull request by daniel-jodlos**
- Handle VAD notification, add `onVadNotification` callback
- Fix onVadNotification callback

### CODE REVIEW

File: src/membraneWebRTC.ts [hunk 1]
```diff
@@ -198,12 +200,26 @@ export interface Callbacks {
...
   onRemoved?: (reason: string) => void;
+
+  /**
+   * Called every time an update about voice activity is received from the server
+   * `vad_status` has two possible values: "silence" and "speech"
+   */
+  onVadNotification?: (
+    peerId: string,
+    trackId: string,
+    vadStatus: "speech" | "silence"
```
Comment (mickel8) [inline src/membraneWebRTC.ts:221]: I would call this `onVoiceActivityChanged` or `onTrackVoiceActivityChanged` (we have similar event in the RTC Engine and it fits better `onTrackEncodingChanged`) . 

What about introducing a separate type for `vadStatus` similar to TrackEncoding? If we decide to do this, please ensure it is correctly displayed in the documentation

**Review by mickel8 — COMMENTED** [review]
As we finally formatted the code it would be nice to enforce formatting in the CI pipeline

**Comment by daniel-jodlos** [comment]
> As we finally formatted the code it would be nice to enforce formatting in the CI pipeline
Sounds like something that should appear in Jira.

**Review by mickel8 — APPROVED** [review]

**Comment by mickel8** [comment]
Created RTC-51

**Pull request merged**